### PR TITLE
Update v0.3 readme [CI SKIP]

### PR DESCRIPTION
**Pull request by guangbochen**
Update v0.3 readme
https://github.com/harvester/harvester/issues/1116

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,58 +19,61 @@ Here are some notable features of the Harvester:
 1. Distributed block storage
 1. Multiple NICs in the VM connecting to the management network or VLANs
 1. Virtual Machine and cloud-init templates
-1. [Rancher](https://github.com/rancher/rancher) integration and the Harvester node driver
+1. [Rancher](https://github.com/rancher/rancher) integration with multi-cluster management and the Harvester node driver
 1. [PXE/iPXE boot support](https://docs.harvesterhci.io/latest/install/pxe-boot-install)
+1. Virtual IP and bond NIC support
+1. Monitoring enhancement
```
Comment (gitlawr) [inline README.md]: Remove enhancement

File: README.md [hunk 2]
```diff
@@ -19,58 +19,61 @@ Here are some notable features of the Harvester:
...
 
-The following diagram gives a high-level architecture of Harvester:
+The following diagram gives a high-level architecture of the Harvester:
```
Comment (gitlawr) [inline README.md]: Remove `the`

File: README.md [hunk 3]
```diff
@@ -19,58 +19,61 @@ Here are some notable features of the Harvester:
...
+- [Elemental for openSUSE Leap 15.3](https://github.com/rancher-sandbox/cOS-toolkit) is a Linux distribution designed to remove as much OS maintenance as possible in a Kubernetes cluster.
 
 ## Hardware Requirements
 To get the Harvester server up and running the following minimum hardware is required:
 
 | Type | Requirements |
 |:---|:---|
-| CPU | x86_64 only. Hardware assisted virtualization required. 4 cores minimum, 16 cores or above preferred |
-| Memory | 8 GB minimum, 32 GB or above preferred |
+| CPU | x86_64 only. Hardware-assisted virtualization is required. 8 cores minimum, 12 cores or above preferred |
```
Comment (gitlawr) [inline README.md]: keep the preferred 16 cores

File: README.md [hunk 4]
```diff
@@ -19,58 +19,61 @@ Here are some notable features of the Harvester:
...
+| Memory | 32 GB minimum, 64 GB or above preferred |
 | Disk Capacity |  120 GB minimum, 500 GB or above preferred |
 | Disk Performance |  5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be [fast enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd). |
 | Network Card | 1 Gbps Ethernet minimum, 10Gbps Ethernet recommended |
 | Network Switch | Trunking of ports required for VLAN support |
 
 We recommend server-class hardware for best results. Laptops and nested virtualization are not supported. 
 
-## Quick start
+## Quickstart
```
Comment (gitlawr) [inline README.md]: `Quick Start` seems more common

File: README.md [hunk 5]
```diff
@@ -19,58 +19,61 @@ Here are some notable features of the Harvester:
...
-Note: This [video](https://youtu.be/97ADieBX6bE) shows a quick overview of the ISO installation.
+During the installation, you can either choose to form a new cluster, or join the node to an existing cluster.
 
 1. Mount the Harvester ISO disk and boot the server by selecting the `Harvester Installer`.
 ![iso-install.png](./docs/assets/iso-install.png)
 1. Choose the installation mode by either creating a new Harvester cluster, or by joining an existing one.
 1. Choose the installation device that the Harvester will be formatted to.
-1. Configure the hostname and select the network interface for the management network, the IP address can either be configured via DHCP or static method.
-![iso-installed.png](./docs/assets/iso-nic-config.png)
+1. Configure the hostname and select the network interface for the management network, by default harvester will create a default bond NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or static method.
```
Comment (gitlawr) [inline README.md]: s/a default bond NIC/a bond NIC/

File: README.md [hunk 6]
```diff
@@ -19,58 +19,61 @@ Here are some notable features of the Harvester:
...
+![iso-installed.png](./docs/assets/iso-nic-config.gif)
+1. (Optional) Config the DNS server and can be separated by the comma.
```
Comment (gitlawr) [inline README.md]: Suggestion:
Configure the DNS servers. Use commas as a delimiter.

File: README.md [hunk 7]
```diff
@@ -19,58 +19,61 @@ Here are some notable features of the Harvester:
...
+1. (Optional) You can choose to import SSH keys from a remote URL server. Your GitHub public keys can be used with `https://github.com/<username>.keys`.
 1. (Optional) If you need to customize the host with cloud-init config, enter the HTTP URL here.
 1. Confirm the installation options and the Harvester will be installed to your host. The installation may take a few minutes to be complete.
 1. Once the installation is complete it will restart the host and a console UI with management URL and status will be displayed. <small>(You can Use F12 to switch between Harvester console and the Shell)</small>
-1. The default URL of the web interface is `https://your-host-ip:30443`.
+1. The default URL of the web interface is `https://your-virtual-ip`.
 ![iso-installed.png](./docs/assets/iso-installed.png)
-1. User will be prompted to set the password for the default `admin` user on the first-time login.
-![first-login.png](./docs/assets/first-log-in.png)
+15. User will be prompted to set the password for the default `admin` user on the first-time login.
```
Comment (gitlawr) [inline README.md]: nit: s/15/1/

**Comment by guangbochen** [comment]
Updated, thanks

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by yasker — APPROVED** [review]

**Pull request merged**